README.md

```diff
@@ -54,13 +54,13 @@ For more information, refer to the [Generative AI Example releases](https://gith
 
 Experience NVIDIA RAG Pipelines with just a few steps!
 
-1. Get your NVIDIA API key.
+1. Get your NVIDIA NGC API key.
    1. Go to the [NVIDIA API Catalog](https://build.ngc.nvidia.com/explore/).
    1. Select any model.
    1. Click **Get API Key**.
    1. Run:
       ```console
-      export NVIDIA_API_KEY=nvapi-...
+      export NGC_API_KEY=nvapi-...
```

Comment:
This should not be needed. Can you provide the exact error you are seeing without setting this?